.changeset/changelog-stylelint.js

```diff
@@ -5,9 +5,44 @@
 const { getInfo, getInfoFromPullRequest } = require('@changesets/get-github-info');
 
 /**
- * @type {import('@changesets/types').ChangelogFunctions}
+ * @type {Array<string | undefined>}
+ */
+const CHANGESET_SECTIONS = ['Deprecated', 'Removed', 'Changed', 'Added', 'Fixed'];
```

Comment:
[suggest] Please use the importance order described in the [doc](https://stylelint.io/maintainer-guide/pull-requests#merging):

> Prefix the item with either: "Removed", "Changed", "Deprecated", "Added", or "Fixed".

```suggestion
const CHANGESET_SECTIONS = ['Removed', 'Changed', 'Deprecated', 'Added', 'Fixed'];
```